src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -106,15 +116,37 @@
 	private FrameLayout mLeftPane;
 	private FrameLayout mRightPane;
 
+	private TabLayout mTabLayout;
```

Comment:
The tabs at the top ("Front Page" and "All" in the screen capture) use up vertical space, and I don't think they're that useful (particularly if the drawer could just include these two items)